test/Conversion/TorchOnnxToTorch/simple_ops_g_to_p.mlir

```diff
@@ -160,6 +162,71 @@ func.func @test_matmul_4d(%arg0: !torch.vtensor<[1,2,3,4],f32>, %arg1: !torch.vt
 
 // -----
 
+// CHECK-LABEL:   func.func @test_maxpool_2d_default(
+// CHECK-SAME:                                       %[[VAL_0:.*]]: !torch.vtensor<[1,3,32,32],f32>) -> !torch.vtensor<[1,3,31,31],f32> attributes {torch.onnx_meta.ir_version = 7 : si64, torch.onnx_meta.opset_version = 12 : si64, torch.onnx_meta.producer_name = "backend-test", torch.onnx_meta.producer_version = ""} {
```

Comment:
For `lit` tests we want to remove extraneous details wherever possible, ie. this check should be relatively minimal:
```
// CHECK-SAME: %[[VAL_0:.*]]: !torch.vtensor<[1,3,32,32],f32>
```

As the rest remains unchanged.